### PR TITLE
Design: Improved Low visibility and contrast in sidebar

### LINKED ISSUE #2367: Design: Low visibility/contrast in sidebar on backstage.io/docs
## Expected Behavior

The sidebar on the Backstage docs page should have a clear visibility of the sections and subsections (contrast and font size).

## Current Behavior

It is currently hard to read the sidebar on the docs page. See screenshot.

low_visibility

## Possible Solution

We can take inspiration from TechDocs page's sidebar.

Screenshot 2020-09-09 at 14 25 45

## Steps to Reproduce

Visit https://backstage.io/docs/features/techdocs/techdocs-overview (maybe disable the darkreader extension :P).

## Your Environment

- NodeJS Version (v12):
- Operating System and Version (e.g. Ubuntu 14.04):
- Browser Information:

### PR DESCRIPTION
**Pull request by iamrajiv**
## Hey, I just made a Pull Request!

Fixes: #2367

Screenshot 2020-09-12 at 2 56 31 AM

#### :heavy_check_mark: Checklist

- [X] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [X] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Comment by OrkoHunter** [comment]
Hey @iamrajiv ! Thanks. Screenshots in the PR would be very helpful. 💯

**Comment by iamrajiv** [comment]
@OrkoHunter I have added the screenshots but I am not able to change the colour of subsection since I did not find the attribute for subsection in `siteConfig.js`. Can you help me with this?

**Comment by OrkoHunter** [comment]
I think you will have to use custom CSS to change the color for the subsection.
See at https://backstage.io/docs/ to find the correct class names for them. And then you can set the color from here https://github.com/spotify/backstage/blob/master/microsite/static/css/custom.css
Hope this helps. :)

**Comment by iamrajiv** [comment]
@OrkoHunter Please take a look.

File: microsite/static/css/custom.css [hunk 1]
```diff
@@ -1017,4 +1021,4 @@ code {
   img {
     margin: auto;
   }
-}
+}
```
Comment (OrkoHunter) [inline microsite/static/css/custom.css:1024]:
```suggestion
}

```

**Review by OrkoHunter — APPROVED** [review]
Looks much better than current. Awesome @iamrajiv ! :D

**Comment by iamrajiv** [comment]
Thank you for guidance @OrkoHunter

**Review by stefanalund — APPROVED** [review]

**Pull request merged**